server.js

```diff
@@ -110,9 +109,20 @@ console.log("databaseOptions is: " + JSON.stringify(serverConfig.databaseOptions
 const parseServer = new ParseServer(serverConfig);
 
 app.get('/dummy.js', function (req, res) {
-  res.type('application/javascript').send("console.log('hello from the dummy service worker');");
+  res.type('text/javascript').send("console.log('hello from the dummy service worker');");
 })
 
+// Note: this endpoint triggers AHOPE application-specific initialization.
```

Comment:
Temporary solution until https://github.com/evandana/keep-ahope/issues/117 is resolved